### DISCUSSION TITLE
Will AlDente support macOS 12 ?

### DISCUSSION THREAD

**Discussion by e-maximilian** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#380] [Q&A]
Will AlDente support macOS 12 ?

**Comment by MatthiasKerbl** [comment]
> Will AlDente support macOS 12 ?

Hi @e-maximilian,

We are working on AlDente supporting macOS 12.

Cheers,
Matthias

**Reply by e-maximilian** [reply]
Hi, macOS 12 will be released soon.
How's the progress of macOS 12 compatibility?

**Reply by MatthiasKerbl** [reply]
Hi @e-maximilian,

The progress regarding the compatibility of AlDente and macOS 12 looks good.
We are currently testing the newest beta version of AlDente with macOS 12 and will release it when it is ready but definitely before macOS 12 will be released.

Cheers,
Matthias

**Reply by e-maximilian** [reply]
Thanks mate.

**Reply by elihollander** [reply]
Any updates on MacOS 12 and AlDente?

**Reply by MatthiasKerbl** [reply]
Hi @elihollander,

We released a new version a couple of hours ago which supports macOS 12 completely.

Cheers,
Matthias

**Comment by elihollander** [comment]
Matthias,

I really appreciate your update and the commitment to AlDente — which is a great application, by the way. And thanks for letting me know. I updated and installed the new OS… all good.

Thanks again,
Eli

_____________________________________________________________
Eli Hollander, Professor Emeritus 831.464.0616 (voice)
Film and Digital Media Department
University of California, Santa Cruz
Santa Cruz, CA 95064
***@***.***
http://film.ucsc.edu/

_____________________________________________________________
